charts/posthog/templates/web-deployment.yaml

```diff
@@ -7,6 +7,12 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+  annotations:
```

Comment:
Confused - why does the web deployment + workers need this exactly?

Reply:
The migrations never were triggered on DigitalOcean. It could have been related to the `atomic` flag on helm. Basically with the `post-install` hook was waiting for `web` and `plugin` deployments to start, which wouldn't because the migrations never ran. This stages it so that as soon as the rest of the stack is ready: Run migrations, and then start plugins and web deployments. In that order.